src/core/lib/security/transport/client_auth_filter.cc

```diff
@@ -429,3 +430,8 @@ const grpc_channel_filter grpc_client_auth_filter = {
     destroy_channel_elem,
     grpc_channel_next_get_info,
     "client-auth"};
+
```

Comment:
please remove all these new empty lines

Reply:
Sorry, pushed a commit to the wrong PR (and OWNERS added you)